optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -34,6 +34,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-test-util</artifactId>
```

Comment:
There is another copy in jbpm, but it has few unique properties. it would be better to rewrite whole PersistenceUtil class.